### PR TITLE
Add Support for State of the DApps

### PR DESCRIPTION
**Pull request by themacexpert**
### Description
This PR adds support for State of the Dapps including a tutorial for how to register. This PR also removes the template for adding a PR to the Moonbeam Project Directory. Existing projects that have already been submitted to the Moonbeam Project directory will continue to live there but new inquiries or PRs will be referred to State of the DApps.

### Checklist

- [x] Translations: The Moonbeam project directory is currently not translated. 

### Corresponding PRs

Please link to any corresponding PRs here.

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: list-your-project.md [hunk 1]
```diff
@@ -7,80 +7,51 @@ description: How you can list your TestNet project or DApp on the Moonbeam docum
...
 
-In general, to be considered and added to this list, your project/DApp must meet the following requirements in terms of content:
+State of the DApps maintains a rank of projects based on active users, transaction volume, developer activity, and profile strength. Moonriver is live on State of the DApps and at the time of writing, both the 2nd and 7th top ranked projects across all ecosystems were built on Moonriver. Previously, a crowd-sourced DApps Listing was maintained [here](/learn/dapps-list/) as part of the Moonbeam docs site. That registry is deprecated and replaced by State of the DApps. 
 
- - Introduction (see template file)
- - Show working contracts and/or front-ends deployed or connected to the Moonbase Alpha TestNet
- - Explain to users how they can test or integrate your project/DApp
- - Link the GitHub repos of the code
- - Link to communication channels
+![Template banner image](images/stateofthedapps/stateofthedapps1.png)
```
Comment (eshaben) [inline list-your-project.md:16]: can you please use kebab case for the file names (ie. `state-of-the-dapps/...`) and update the image description

File: list-your-project.md [hunk 2]
```diff
@@ -7,80 +7,51 @@ description: How you can list your TestNet project or DApp on the Moonbeam docum
...
 
-The main idea is to fork a repository, modify it with your changes, and then submit a PR.
+At a minimum, you must include the following information to submit your project/DApp to State of the DApps:
 
-So, as mentioned before, first fork [this repository](https://github.com/PureStake/moonbeam-project-directory).
+ - DApp Name
+ - One-Sentence Description
+ - Full Description
+ - Website URL
+ - Life cycle Status
```
Comment (eshaben) [inline list-your-project.md:31]: maybe capitalize cycle to be consistent

File: list-your-project.md [hunk 3]
```diff
@@ -7,80 +7,51 @@ description: How you can list your TestNet project or DApp on the Moonbeam docum
...
+ 8. Provide a logo, icon, and screenshots if applicable
+ 9. Select your DApp’s status
+ 10. Add your project's social media links if applicable
+ 11. Choose a category
+ 12. Provide at least one tag for your DApp. If your desired tag hasn’t been created yet, you can create a new tag
+ 13. Provide a contact email (this is not shared publicly)
+ 14. Review the terms and conditions and press **Submit**
 
-For example, let's say your project is named "Rocket Project" and related to DeFi. Then, you would need to copy this file inside the following folder:
+![How to Submit your DApp](images/stateofthedapps/stateofthedapps2.png)
```
Comment (eshaben) [inline list-your-project.md:55]: same comment here about using kebab case for the file name

**Review by eshaben — CHANGES_REQUESTED** [review]
This looks good! Just a few minor changes. Also, since the `template.md` page has been removed, the banner image and the index page icon for that page can be removed too

**Comment by themacexpert** [comment]
Thanks for the feedback @eshaben I implemented all changes, the only thing remaining is for the index page icon to be removed but that needs to be removed from the Moonbeam Docs repo so I will take care of that in a future PR.

**Pull request merged**